core/nodepool/config/templates/cloud-config-worker

```diff
@@ -481,7 +481,7 @@ coreos:
         --container-runtime={{.ContainerRuntime}} \
         --rkt-path=/usr/bin/rkt \
         --rkt-stage1-image=coreos.com/rkt/stage1-coreos \
-        --node-labels=kubernetes.io/role=node{{if .NodeLabels.Enabled}},{{.NodeLabels.String}}{{end}} \
+        --node-labels=node-role.kubernetes.io/node="",kubernetes.io/role={{ toLabel .NodePoolName }}{{if .NodeLabels.Enabled}},{{.NodeLabels.String}}{{end}} \
```

Comment:
Shouldn't this be:
```
--node-labels=kubernetes.io/role=node,node-role.kubernetes.io/node="",node-role.kubernetes.io/{{ toLabel .NodePoolName }}=""{{if .NodeLabels.Enabled}},{{.NodeLabels.String}}{{end}}
```
so that the `kubernetes.io/role/node` remains backward-compatible, while we could still use `node-role.kubernetes.io/whatever` to assign multiple roles per a node?

Ref: https://github.com/kubernetes/kubernetes/pull/51173/files#diff-749b0f5911ddd5261b4ed5c2945d7c16R1202

Reply:
Added b646541 for it. Please feel free to comment/revert/approve/etc 😃